### PR TITLE
Appveyor

### PR DESCRIPTION
**Pull request by mirko-pagliai**

### CODE REVIEW

**Comment by mirko-pagliai** [comment]
@swiffer for now the code works with mysql and postgre (except for some tests to be fixed, but it is not a problem).
Instead I can not install the sqlite3 executable.

File: appveyor.yml [hunk 1]
```diff
@@ -43,6 +43,7 @@ install:
   - cd c:\
   - appveyor DownloadFile https://www.sqlite.org/2018/sqlite-tools-win32-x86-3220000.zip -Filename sqlite3.zip
   - 7z x sqlite3.zip -oc:\sqlite3 > nul
+  - rename sqlite-tools-win32-x86-3220000 sqlite3
```
[the remark this answers is not in the document]
Comment (swiffer) [reply appveyor.yml]: yes, binaries where in subfolder named like zip file -.-

**Comment by mirko-pagliai** [comment]
now the problem is to delete the sqlite database file, it seems to be blocked.
This happens when a backup is imported: the current database must be deleted and recreated.

**Comment by swiffer** [comment]
i think it is blocked because you open a second connection to the same file within ``tests\TestCase\Driver\SqliteTest.php`

**Comment by mirko-pagliai** [comment]
I've already solved that :-)

last build: https://ci.appveyor.com/project/mirko-pagliai/cakephp-database-backup/build/1.0.62

**Comment by mirko-pagliai** [comment]
I've already solved that :-)

last build: https://ci.appveyor.com/project/mirko-pagliai/cakephp-database-backup/build/1.0.63

**Comment by mirko-pagliai** [comment]
Now all works. But I want to understand these warnings:

```
pg_dump: [custom archiver] WARNING: ftell mismatch with expected position -- ftell used
pg_dump: [custom archiver] WARNING: ftell mismatch with expected position -- ftell used
pg_dump: [custom archiver] WARNING: ftell mismatch with expected position -- ftell used
pg_dump: [custom archiver] WARNING: ftell mismatch with expected position -- ftell used
pg_dump: [custom archiver] WARNING: ftell mismatch with expected position -- ftell used
pg_dump: [custom archiver] WARNING: ftell mismatch with expected position -- ftell used
```

**Comment by mirko-pagliai** [comment]
This seems [a problem with the pipe notation](https://www.postgresql.org/message-id/23854.1446474615%40sss.pgh.pa.us).

I think that we can go on anyway, it's just a warning, but the recovery happens correctly

**Comment by mirko-pagliai** [comment]
Finally merged into master ;) Great work!

**Comment by swiffer** [comment]
wow! thanks for your great work!

**Comment by mirko-pagliai** [comment]
just a few minutes for the last tasks

**Comment by mirko-pagliai** [comment]
Released. Bye!

**Pull request merged**